themes/nuboot_radix/theme-settings.php

```diff
@@ -81,7 +81,32 @@ function nuboot_radix_form_system_theme_settings_alter(&$form, &$form_state) {
     ),
   );
 
+  $form['site_information'] = array(
```

Comment:
@jacintocapote please add some inline comments that explain what this is here for.